packages/stream_chat_flutter/lib/src/poll/stream_poll_text_field.dart

```diff
@@ -3,12 +3,12 @@ import 'package:stream_chat_flutter/stream_chat_flutter.dart';
 
 const _kTransitionDuration = Duration(milliseconds: 167);
 
-/// {@template pollInputTextField}
+/// {@template streamPollTextField}
 /// A widget that represents a text field for poll input.
 /// {@endtemplate}
-class PollTextField extends StatefulWidget {
-  /// {@macro pollInputTextField}
-  const PollTextField({
+class StreamPollTextField extends StatefulWidget {
+  /// {@macro streamPollTextField}
+  const StreamPollTextField({
```

Comment:
Is this a breaking change for integrators? 
- did we mention this in the changelog?
- did we apply renames to the docs?

Reply:
Poll isn't released yet